Application/EdFi.Ods.Common/Constants/ApiFeature.cs

```diff
@@ -16,6 +16,7 @@ public class ApiFeature : Enumeration<ApiFeature, string>
         public static readonly ApiFeature OwnershipBasedAuthorization = new ApiFeature("ownershipBasedAuthorization", "Ownership Based Authorization");
         public static readonly ApiFeature UniqueIdValidation = new ApiFeature("uniqueIdValidation", "Unique Identification Validation");
         public static readonly ApiFeature TokenInfo = new ApiFeature("tokenInfo", "Token Introspective Endpoint");
+        public static readonly ApiFeature IdentityManagement = new ApiFeature("identityManagement", "Identity Management");
```

Comment:
You've added the new Feature, but none of the other supporting code seems to be here.  Missing an AutoFac module for conditionally setting up this feature (registering the services needed), and I would expect a PR against the Implementation repo as well for config changes as well as powershell updates to turn it on by default for development.